docs/api-reference/base-layer.md

```diff
@@ -82,10 +86,10 @@ The opacity of the layer. deck.gl automatically applies gamma to the opacity
 in an attempt to make opacity changes appear linear (i.e. the opacity is
 visually proportional to the value of the prop)
 
-Remarks
-* While it is a recommended convention that all deck.gl layers should
-  support the `opacity` prop, it is up to each layer's fragment shader
-  to implement support for opacity.
+**Remark:**
```

Comment:
Nit: Fine, we have a pretty consistent style of having bulleted remarks, but this works too.